libs/components/src/lib/text-anchor/text-anchor.template.ts

```diff
@@ -5,13 +5,13 @@ import type {
 	FoundationElementDefinition,
 } from '@microsoft/fast-foundation';
 import { classNames } from '@microsoft/fast-web-utilities';
-import { affixIconTemplateFactory } from '../../shared/patterns/affix';
+import { affixIconTemplateFactory, IconWrapper } from '../../shared/patterns/affix';
 import { focusTemplateFactory } from './../../shared/patterns/focus';
 import type { TextAnchor } from './text-anchor';
 
 const getClasses = ({text, icon}: TextAnchor) => classNames(
 	'control',
-	['icon-only', !text && !!icon],
+	['icon-only', (!text && !!icon) || (!text && !!IconWrapper.Span)],
```

Comment:
What does this logic do? `!!IconWrapper.Span` is always going to be true

So it's equivalent to this?
```suggestion
	['icon-only', !text],
```